docs/aca/07-aca-cron-bindings/index.md

```diff
@@ -92,50 +105,120 @@ public interface ITasksManager
     Task<List<TaskModel>> GetYesterdaysDueTasks();
 }
 ```
-Next we need to provide implementation for those 2 methods, so open file `TasksStoreManager.cs` in the same project and use the code below:
+Next we need to provide the implementation for those two methods. Open the file `TasksStoreManager.cs` which is located under the `services` folder of the `TasksTracker.TasksManager.Backend.Api` project and place the two following methods inside the existing class:
 
 ```csharp
-public class TasksStoreManager : ITasksManager
+public async Task<List<TaskModel>> GetYesterdaysDueTasks()
+{
+    var options = new JsonSerializerOptions
+    {
+        PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+        WriteIndented = true,
+        Converters =
+        {
+            new JsonStringEnumConverter(),
+            new DateTimeConverter("yyyy-MM-ddTHH:mm:ss")
+        },
+        PropertyNameCaseInsensitive = true,
+        DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+        Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
+    };
+    var yesterday = DateTime.Today.AddDays(-1);
+
+    var jsonDate = JsonSerializer.Serialize(yesterday, options);
```

Comment:
@waelkdouh Were you able to test that returned datetime format matches the one used with Newtonsoft and execute the method successfully? Let's make sure that at least some tasks are marked overdue after this change. From the code here it should work as you are specifying the format explicitly.

Reply:
@tjoudeh can you please take care of this one as I don't have a working version of the Newtonsoft implementation. I will go ahead as this is not a blocking change. Thanks